src/QueryRunner.php

```diff
@@ -64,12 +64,6 @@ private function processScript(Block $block, Code $code, Script $script): void
             return;
         }
 
-        // Skip select
-        if (strtoupper(substr($sql, 0, 6)) === 'SELECT') {
```

Comment:
@ondrajodas hm, nemozes dat ten `SELECT` do zatvorky `(SELECT` .... lebo toto skipovanie SELECT queries bolo priamo v zadani pre transformacie -> ze casto krat si niekto vytvori transformaciu z workspace a je tam kopec selectov, alebo take nieco ...

Reply:
pripadne tu hod query co ti to blokuje a mozem tiez pozriet ...

Reply:
je v komentáři

Reply:
možná by šlo přeskakovat jen query co mají SELECT a nemají INTO

Reply:
sorry, nevsimol som si ... jj to `INTO` dava zmysel, asi by som iba v v SQLku hladal ` INTO `, pri najhorsom to spusti select, ... napr. ak tam bude `SELECT " INTO " AS TXT` , ale vacsinu situacii to vyriesi.

Reply:
ještě koukám na travisu chybí spousta envů, ty je tam vždy doplňuješ?

Reply:
už to vidím, oni se nám měnily subskripce
```
ERROR: No subscriptions were found for 'ca539ce0-c582-4301-8cd0-a90af3720017'. If this is expected, use '--allow-no-subscriptions' to have tenant level accesses
```

Reply:
@webrouse uměl by si to fixnout? mě by to trvalo asi dlouho než bych to našel všechno ^

Reply:
ono tam je script co vytvara synapse ... jj, treba vytvorit service account v nasej subkripcii ... synapse sa v CI potom uz vytvori automaticky ... mam na to task, ku ktoremu som sa v backlogu este nedostal

Reply:
aj :( to je blbý, bez toho se nepohneme dál :/ leda pozastavit testy

Reply:
@ondrajodas tu som nasiel nejaky navod, nam treba vytvorit iba toho service principal: https://keboola.atlassian.net/wiki/spaces/TECH/pages/340328508/Synapse+for+Dev#Create%2FDelete-Synapse-server-in-CI%3A

Potrebujes mat nainstalovane `azure cli` .... potom das `az login` a prihlasis sa a ides podla navodu.
Da sa to cele skusat lokalne, ... ked budes mat service principal, tak moze lokalne vyskusat vytvorit synapse -> a potom by to malo ist aj na CI.

Reply:
asi bych to udělal zatím takto....potřebuju dotáhnout ty transformace a s tímto si můžeme hrát asi potom jestli s tím takto souhlasíš?

Reply:
ok, cas nas tlaci